src/client/FlagsClient.test.ts

```diff
@@ -5,6 +5,7 @@ describe('FlagsClient', () => {
     appName: 'production',
     instanceId: 'foo',
     url: 'https://foo.bar/api',
+    uri: '/client/features',
```

Comment:
If you also test with a custom uri, and not the fallback, you can validate the uri has been passed correctly.

Reply:
Thanks added test!